### PR TITLE
Add 'brackets-minify' extension

### PR DESCRIPTION
**Pull request by HunseopJeong**
- To support minify feature, add 'brackets-minify' extension.
- Move menu divider from 'hirse.beautify' to the 'brackets-minify'

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: libs/brackets-server/embedded-ext/brackets-minify/node/minifyjs.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+(function() {
+    "use strict";
+
+    var path     = require('path'),
+        fs       = require('fs'),
+        mkpath   = require('mkpath'),
+        UglifyJS = require("uglify-js");
+
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/brackets-minify/node/minifyjs.js]: single qutation and double quatation mark seems to be mixed on same file.
coding style ?

File: libs/brackets-server/embedded-ext/brackets-minify/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "description": "Easily minify single files or all files in a project. Also includes concatenation features as well as auto-minification on save.",
+    "homepage": "https://github.com/abagshaw/brackets-minifier",
+    "version": "2.0.0",
+    "author": "Andrew Bagshaw",
+    "license": "https://github.com/abagshaw/brackets-minifier#license",
+    "engines": {
+        "brackets": ">=0.24.0"
+    },
+	"i18n": [
+        "en"
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/brackets-minify/package.json]: no support for korean ?

File: libs/brackets-server/embedded-ext/brackets-minify/main.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
+define(function(require, exports, module) {
+    "use strict";
+
+    var CommandManager  = brackets.getModule("command/CommandManager"),
+        Menus           = brackets.getModule("command/Menus"),
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/brackets-minify/main.js]: ordering ?

File: libs/brackets-server/embedded-ext/brackets-minify/node/minifyjs.js (hunk 1 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/brackets-minify/node/minifyjs.js]: Fixed

File: libs/brackets-server/embedded-ext/brackets-minify/package.json (hunk 2 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/brackets-minify/package.json]: Done.

File: libs/brackets-server/embedded-ext/brackets-minify/main.js (hunk 3 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/brackets-minify/main.js]: Done.

**Pull request merged**